src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,83 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] text = openFile(fromFileName);
+        int[] data = calcutData(text);
+        String[] dataForPrint = printData(data);
+        putDataToFile(dataForPrint,toFileName);
+    }
+
+    private String[] openFile(String fileName) {
+        StringBuilder string = new StringBuilder();
+        try {
+            File file = new File(fileName);
+            FileReader fileReader = new FileReader(file);
+            BufferedReader bufferedReader = new BufferedReader(fileReader);
```

Comment:
do you close resources?

Reply:
not fixed, we should either use try with resources or finally block for this
check [this](https://stackoverflow.com/questions/56151113/use-try-with-resources-or-close-this-bufferedreader-in-a-finally-clause) for details